Documentation/teaching/lectures/debugging.rst

```diff
@@ -612,14 +612,6 @@ KASan vs SLUB_DEBUG
      allocation/freeing of object. KASan catch bugs right before it will happen,
      so we always know exact place of first bad read/write.
 
-  * SLUB_DEBUG has lower overhead than KASan.
```

Comment:
Not sure what exactly is this commit here, but it doesn't look good. Please always rebase your changes on top of latest master branch.

Reply:
I'm not sure either. Master does not have that line either.
I'll undo this...somehow.